src/main/java/com/laytonsmith/core/functions/Echoes.java

```diff
@@ -836,6 +842,19 @@ public Mixed exec(Target t, Environment environment, Mixed... args) throws Confi
 							b.append(color.exec(t, environment, new CString(c, t)));
 							i += sl;
 						} else {
+							if (c.equals('#')) {
```

Comment:
Build error:
`src\main\java\com\laytonsmith\core\functions\Echoes.java:845: Whitespace between if/for/while and ( is not allowed. [RegexpSingleline]`.